### PR TITLE
chore: expand label-actions about one topic and no mentions

### PR DESCRIPTION
**Pull request by rarkins**
## Changes

Adds two new label actions

## Context

These are frequent problems we should automate

## Documentation (please check one with an [x])

- [ ] I have updated the documentation, or
- [x] No documentation update is required

## How I've tested my work (please select one)

I have verified these changes via:

- [x] Code inspection only, or
- [ ] Newly added/modified unit tests, or
- [ ] No unit tests but ran on a real repository, or
- [ ] Both unit tests + ran on a real repository

### CODE REVIEW

**Review by viceice — DISMISSED** [review]

File: .github/label-actions.yml [hunk 1]
```diff
@@ -274,3 +274,25 @@
...
 
     Thanks, the Renovate team
+
+'auto:one-topic':
+  comment: >
+    Hi there,
+
+    Please stick to one topic/question per Discussion. Start a new discussion per topic.
+
+    This is to benefit future users who may search and find this discussion and not be confused because there's multiple questions and answers within. One question and answer per Discussion works best.
```
Comment (HonkingGoose) [inline .github/label-actions.yml:284]:
```suggestion
    This prevents future users from finding this discussion and getting confused by multiple questions and answers. One question and answer per Discussion works best.
```

File: .github/label-actions.yml [hunk 2]
```diff
@@ -274,3 +274,25 @@
...
+
+    For example:
+     - Don't create a discussion with multiple questions requiring multiple answers
+     - If you got an answer to your first question, don't change topic like "Can I also ask you how I might..." which would then require a different answer. Mark the current discussion as answered and open a new one.
+
+'auto:no-mentions':
+  comment: >
+    Hi there,
+
+    Please do not unnecessarily `@` maintainers like `@rarkins` or `@viceice`. Doing so causes annoying notifications and make maintaining a repo of this size difficult.
```
Comment (HonkingGoose) [inline .github/label-actions.yml:294]:
```suggestion
    Please do not unnecessarily `@` mention maintainers like `@rarkins` or `@viceice`. Doing so causes annoying notifications and makes it harder to maintain this repository.
```

File: .github/label-actions.yml [hunk 3]
```diff
@@ -274,3 +274,25 @@
...
+
+    For example, never `@` mention a maintainer when you are creating a discussion if your desire is to get attention. This is inconsiderate behavior, just like shouting out your coffee order in a Starbucks before it's your turn.
```
Comment (HonkingGoose) [inline .github/label-actions.yml:296]:
```suggestion
    For example, never `@` mention a maintainer when you are creating a discussion if your desire is to get attention. This is rude behavior, just like shouting out your coffee order in a Starbucks before it's your turn.
```

`Inconsiderate` is a difficult word, I think `rude` or `bad` is easier to understand.

File: .github/label-actions.yml [hunk 4]
```diff
@@ -274,3 +274,25 @@
...
+
+    If there's an existing issue or discussion which you were hoping for a follow-up to but multiple days or weeks have elapsed, it's ok for you to comment, but please still do not `@` people. The maintainers do not promise to answer every Discussion but do answer most. If you're not getting an answer then take a look at the informatio you've provided and see if you can improve it.
```
Comment (HonkingGoose) [inline .github/label-actions.yml:298]:
```suggestion
    It's OK to comment in an issue or discussion after multiple days or weeks. But please, still don't `@` mention people. The maintainers try to answer most discussions, but they can't answer all discussions. If you're still not getting an answer, take a look at the information you've given us and see if you can improve it.
```

The current text is hard to understand, how about this?

**Review by HonkingGoose — CHANGES_REQUESTED** [review]

File: .github/label-actions.yml [hunk 5]
```diff
@@ -274,3 +274,25 @@
 
 
     Thanks, the Renovate team
```
Comment (HonkingGoose) [inline .github/label-actions.yml:276]: The two new comments should have the `Thanks, the Renovate team` ending as well. 😉

**Pull request merged**